Servers/Zeus/ARC/.arc/submit.py

```diff
@@ -14,14 +14,15 @@
 #PBS -o out.txt
 #PBS -e err.txt
 
+. ~/.bashrc
+
 PBS_O_WORKDIR={pwd}
 cd $PBS_O_WORKDIR
 
 GAUSS_SCRDIR=/home/{un}/scratch/g09/$PBS_JOBID
-export $GAUSS_SCRDIR
 mkdir -p $GAUSS_SCRDIR
-
-. ~/.bashrc
+export $GAUSS_SCRDIR
```

Comment:
What is the purpose of this command?
We tried to run line after line on bash and this one returns:
-bash: export: `/home/slily91/scratch/g09/12': **not a valid identifier**

I believe it should be w/o the '$' since you're only supposed to type the name of the variable, and not call its value. That way bash doesn't return an error.